base_layer/wallet_ffi/src/error.rs

```diff
@@ -106,6 +106,10 @@ impl From<WalletError> for LibWalletError {
                 code: 101,
                 message: format!("{:?}", w),
             },
+            WalletError::OutputManagerError(OutputManagerError::FundsPending) => Self {
+                code: 115,
```

Comment:
This is going to set someone's OCD off

Reply:
lol I figured better to keep it next to NotEnoughFunds 🤷‍♂️